src/containers/Create.jsx

```diff
@@ -15,10 +17,96 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export const Create = (props) => {
+  const [members, setMembers] = useState([]);
   const { register, handleSubmit, errors } = useForm();
-  const onSubmit = (data) => console.log(data);
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [firstNameKana, setFirstNameKana] = useState("");
+  const [lastNameKana, setLastNameKana] = useState("");
+  const [githubId, setGithubId] = useState("");
+  const [twitterId, setTwitterId] = useState("");
+  const [repositoryUrl, setRepositoryUrl] = useState("");
+  const [startDate, setStartDate] = useState("");
+  const [memo, setMemo] = useState("");
+  // const [member, setMember] = useState({
+  //   id: 1,
+  //   first_name: "",
+  //   last_name: "",
+  //   first_name_kana: "",
+  //   last_name_kana: "",
+  //   github_id: "",
+  //   twitter_id: "",
+  //   repository_url: "",
+  //   start_date: "",
+  //   memo: "",
+  // });
+
+  const postMember = async (data) => {
+    await axios
+      .post(
+        "http://localhost:4001/api/members",
+        {
+          member: {
+            id: 22,
+            first_name: firstName,
+            last_name: lastName,
+            first_name_kana: firstNameKana,
+            last_name_kana: lastNameKana,
+            github_id: githubId,
+            twitter_id: twitterId,
+            repository_url: repositoryUrl,
+            start_date: startDate,
+            memo: memo,
+          },
+        },
+        { withCredentials: true }
+      )
+      .then((res) => {
+        console.log("data", data);
+        console.log("res data", res);
+        console.log("res data", res.data.first_name);
+
+        setMembers([
+          ...members,
+          {
+            id: 11,
+            first_name: res.data.first_name,
+            last_name: res.data.last_name,
+            first_name_kana: res.data.first_name_kana,
+            last_name_kana: res.data.last_name_kana,
+            github_id: res.data.github_id,
+            repository_url: res.data.repository_url,
+            twitter_id: res.data.twitter_id,
+            start_date: res.data.start_date,
+            memo: res.data.memo,
+          },
+        ]);
+      })
+      .catch((error) => {
+        console.log("registration error", error);
+      })
+      .catch((data) => {
+        console.log(data);
+      });
+  };
```

Comment:
少しまだ非同期処理の理解が、浅いかもしれないので特訓したいですね
せっかくasync functionで書くのであればこんな感じですかね

```suggestion
  const postMember = async (data) => {
    const res = await axios
      .post(
        "http://localhost:4001/api/members",
        {
          member: {
            id: 22,
            first_name: firstName,
            last_name: lastName,
            first_name_kana: firstNameKana,
            last_name_kana: lastNameKana,
            github_id: githubId,
            twitter_id: twitterId,
            repository_url: repositoryUrl,
            start_date: startDate,
            memo: memo,
          },
        },
        { withCredentials: true }
      )
  };
```

Reply:
こんな感じで書くと`res` には リクエストに成功すれば200番のレスポンスが、失敗したら500番のレスポンスが入っているはずです。

`catch` でエラー拾ってると思うんですけど、これは前ペアプロした時みたいにtry catch側で拾ってあげるといいですかねー